src/operator/contrib/nms.cc

```diff
@@ -0,0 +1,84 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file nms.cc
+ * \brief Non-maximum suppression for bounding boxes, cpu operator
+ * \author Joshua Zhang
+ */
+#include "./nms-inl.h"
+#include "../elemwise_op_common.h"
+
+namespace mxnet {
+namespace op {
+DMLC_REGISTER_PARAMETER(NMSParam);
+
+NNVM_REGISTER_OP(_contrib_nms)
```

Comment:
we probably should call this something more specific to object detection. Generic nms usually deals with pixels.